ttnn/cpp/ttnn/operations/moreh/moreh_getitem/device/moreh_getitem_device_operation.hpp

```diff
@@ -0,0 +1,100 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+#include <optional>
+#include <variant>
+#include <vector>
+
+#include "ttnn/decorators.hpp"
+#include "ttnn/device_operation.hpp"
+#include "ttnn/tensor/types.hpp"
+
+namespace ttnn::operations::moreh::moreh_getitem {
+struct MorehGetItemOperation {
+    struct operation_attributes_t {
+        const std::vector<uint32_t> index_dims;
+        // const CoreRange core_range;
+        const MemoryConfig output_mem_config;
```

Comment:
```suggestion
        const MemoryConfig output_memory_config;
```
It seems that TT prefers `memory_config` over `mem_config`

Reply:
I will approve once this is fixed.